pkgs/development/python-modules/dedupe/default.nix

```
@@ -0,0 +1,114 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  cython,
+  setuptools,
+  wheel,
+
+  # dependencies
+  affinegap,
+  btrees,
+  categorical-distance,
+  dedupe-levenshtein-search,
+  doublemetaphone,
+  haversine,
+  highered,
+  numpy,
+  scikit-learn,
+  simplecosine,
+  zope-index,
+
+  # tests
+  pytestCheckHook,
+  python,
+  runCommand,
+}:
+
+let
+  dedupe = buildPythonPackage rec {
```

```suggestion
  buildPythonPackage rec {
```
Add dedupe to arguments instead.

---

Nice